examples/inspection/plot_partial_dependence.py

```diff
@@ -27,10 +27,10 @@
 .. [2] For classification you can think of it as the regression score before
        the link function.
 
-.. [3] Goldstein, A., Kapelner, A., Bleich, J., and Pitkin, E., Peeking Inside
-       the Black Box: Visualizing Statistical Learning With Plots of
-       Individual Conditional Expectation. (2015) Journal of Computational and
-       Graphical Statistics, 24(1): 44-65 (https://arxiv.org/abs/1309.6392)
+.. [3] :arxiv:`Goldstein, A., Kapelner, A., Bleich, J., and Pitkin, E. (2015).
+       "Peeking Inside the Black Box: Visualizing Statistical Learning With Plots of
+       Individual Conditional Expectation". Journal of Computational and
+       Graphical Statistics, 24(1): 44-65 <1309.6392>
```

Comment:
I believe there is a missing 

```suggestion
       Graphical Statistics, 24(1): 44-65 <1309.6392>`
```

You can check the rendering here: https://161680-843222-gh.circle-artifacts.com/0/doc/auto_examples/inspection/plot_partial_dependence.html

More generally for the other files:

https://161680-843222-gh.circle-artifacts.com/0/doc/_changed.html

which is generated for each new commit to this PR. To get the link to the latest rendering, click on the "Details" link for the doc artifact in the Continuous Integration report:

![image](https://user-images.githubusercontent.com/89061/144907047-1297fe5d-909f-44ae-9bd2-7781efc0949c.png)